.github/workflows/helm-checks.yaml

```diff
@@ -147,7 +160,7 @@ jobs:
 
     - name: Install chart for data exchange (umbrella)
       run: |
-        helm install umbrella charts/umbrella -f charts/values-test-data-exchange.yaml -f charts/values-test-iam-init-container.yaml --namespace data-exchange --create-namespace --debug --timeout 10m
+        helm install umbrella charts/umbrella -f charts/values-test-data-exchange.yaml -f charts/values-test-iam-init-container.yaml --namespace data-exchange --create-namespace --debug --timeout 10m --set semantic-hub.graphdb.image=kind-registry:5000/jena-fuseki-docker:4.7.0
```

Comment:
I was wondering if we should have a separate step for the semantic hub, as it is disabled by default anyways. What do you think @evegufy?

Reply:
As discussed, I can be added to the shared services